<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewInterActivityForm</class>
 <widget class="QDialog" name="NewInterActivityForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Interactivity</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Config</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Module   :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>idPrefix  :</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>321</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Flags</string>
   </property>
   <widget class="QCheckBox" name="cb_isNewJsonType">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>isNewJsonType</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_newSaveState">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>newSaveState</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_isSaveStateAllowed">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>isSaveStateAllowed</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_isTwoStepLoad">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>isTwoStepLoad</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_showMute">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>showMute</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>144</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>playerTheme :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>178</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>themeType :</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Model</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>21</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Class :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="modelClassText">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CamelCaseData</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>353</x>
     <y>0</y>
     <width>20</width>
     <height>483</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>421</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>OverViewTab</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>templateName :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="overViewTabTemplateText">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>overview</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>class :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>el :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="overViewTabClass">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Overview</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="overViewTabEL">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>#overview-tab-holder</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>overViewTabData</string>
    </property>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ScreenId :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="overViewScreenId">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>overview-screen</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>leftImageContainerId :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leftImageId">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>overview-bg</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>417</width>
     <height>131</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Tab Title</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>TemplateName</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Class</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>el</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>Tab 1</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>tab-1</string>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>Tab1Class</string>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string>#tab-1-holder</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>Tab 2</string>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string>tab-2</string>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string>Tab2Class</string>
    </property>
   </item>
   <item row="1" column="3">
    <property name="text">
     <string>#tab-2-holder</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>182</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tabs ( other than overview tab )</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addTabBtn">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>180</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Tab</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeTabBtn">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>180</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Tab</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="moduleText">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>76</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>CamelCase</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idPrefixText">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>camel-case</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="playerThemeText">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>145</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="themeTypeText">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>490</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancalBtn">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>490</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Folder Name :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="folderNameText">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ge-u#-c#-i#-interactivity-name</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>folderNameText</tabstop>
  <tabstop>moduleText</tabstop>
  <tabstop>idPrefixText</tabstop>
  <tabstop>playerThemeText</tabstop>
  <tabstop>themeTypeText</tabstop>
  <tabstop>cb_isTwoStepLoad</tabstop>
  <tabstop>cb_isNewJsonType</tabstop>
  <tabstop>cb_newSaveState</tabstop>
  <tabstop>cb_isSaveStateAllowed</tabstop>
  <tabstop>modelClassText</tabstop>
  <tabstop>overViewTabTemplateText</tabstop>
  <tabstop>overViewTabClass</tabstop>
  <tabstop>overViewTabEL</tabstop>
  <tabstop>overViewScreenId</tabstop>
  <tabstop>leftImageId</tabstop>
  <tabstop>addTabBtn</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>removeTabBtn</tabstop>
  <tabstop>saveBtn</tabstop>
  <tabstop>cancalBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
